README.md

```diff
@@ -8,21 +8,84 @@
 
 AKS-Engine leverages ARM (Azure Resource Manager) to build Kubernetes IaaS in Azure. AKS-Engine provides convenient tooling to quickly bootstrap clusters, and implements cluster provisioning and lifecycle operations for [AKS](https://docs.microsoft.com/en-us/azure/aks), Azure's managed Kubernetes service offering.
 
-More info, including a thorough walkthrough is [here](docs/aksengine.md).
+## Install AKS-Engine
 
-Please see the [FAQ](/docs/faq.md) for answers about AKS-Engine and its progenitor ACS-Engine.
+To install via homebrew, run `brew tap azure/aks-engine && brew install azure/aks-engine/aks-engine` or download the binary via the [github releases page](https://github.com/Azure/aks-engine/releases)
```

Comment:
Is there a ticket out there to create this homebrew tap? https://github.com/Azure/homebrew-aks-engine 404s for me, so I assume this is an action item?

Reply:
Yeah, everything here is aspirational! Let's bikeshed the aspirations, and then when we have a good idea of where we want things to land, then we can start tracking TODOs.

Reply:
https://github.com/Azure/aks-engine/issues/227

Reply:
Should add chocolatey too to cover Windows users